tutorials/ui/gui_theme_type_variations.rst

```diff
@@ -0,0 +1,50 @@
+.. _doc_gui_theme_type_variations:
+
+Overview
+========
+
+When designing a user interface there may be times when a control node
+needs to have a different look than what is normally defined by the
+theme. Theme override properties of a node can be used. However if
+this change needs to be done multiple times the same work is being done
+multiple times, it also requires finding and editing every individual
+node with that look if it needs to be adjusted.
+
+As a solution to this problem there are theme type variations. These
+are used to create variations of control nodes in a way that is part
+of the theme.
+
+These can be used to redefine every aspect of a control node or only one.
+For example, if a button has a defined StyleBoxFlat for its normal state in
+the theme, and a button type variation only has a different font specified,
+any button using that type variation will still use the StyleBoxFlat defined
+for the theme.
+
+Creating a type variation
+-------------------------
+
+To create a type variation open the theme editor, then open the **Manage
+Theme Items** window. In the bottom left under **Add Type** enter a name
+for the  type variation then press the add button.
+
+Close the manage theme items window. In the theme editor on the right
+side click on the **Type** dropdown above the type editor. The type created
+in manage theme items should be listed, select it, then switch to the
+tab with a wrench and screwdriver icon.
+
+.. img:: img/base_type.png
+
+Click on the plus icon. This windows allows you to select the control node
+the type variation is going to be used for. After selecting a node the tabs
+for this type variation should now have the same style options as that control
+node and can be edited.
```

Comment:
```suggestion
Click on the plus icon next to the **Base Type** field. You can select the base type
there, which would typically be the name of a control node class (e.g., ``Button``, ``Label``, etc).
Type variations can also chain and extend other type variations. This works in the
same way control nodes inherit styling of their base class. For example, ``CheckButton``
inherits styles from ``Button`` because corresponding node types extend each other.

After you select the base type, you should now be able to see its properties on the other
tabs in the theme editor. You can edit them as usual.
```